### PR TITLE
Project import generated by Copybara.

### PR DESCRIPTION
**Pull request by murilolima**
GitOrigin-RevId: fbe3c78f8f3ea97b07f9ad36c32b7ac60cbc0b8a

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## [Unreleased]
```
Comment (stoman) [inline CHANGELOG:3]: Should we create a new release for this? I think every push to GitHub should generally be a new release and have a new version number.

Comment (murilolima) [reply CHANGELOG:3]: Done.

**Comment by stoman** [comment]
Thank you for running this update. It's great to see that it works well to push to GitHub.

**Review by stoman — APPROVED** [review]
Thank you for running this update. It's great to see that it works well to push to GitHub.

**Pull request merged**